Sources/XiCliCore/XiCliCore.m

```diff
@@ -0,0 +1,19 @@
+// Copyright 2018 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "XiCliCore.h"
+
+@implementation XiCliCore
```

Comment:
are these two files necessary?

Reply:
they may well be required for a static lib or something, I'm not sure.

Reply:
Once I fixed the test target linker error I was able to delete them.